backend/src/api/permissions/permissions.api.ts

```diff
@@ -18,8 +24,16 @@ const initPermissionsApi: FastifyPluginAsync<Options> = async (
   fastify.route({
     method: HttpMethod.GET,
     url: PermissionApiPath.ROOT,
-    async handler(req, rep) {
-      const permissions = await permissionService.getAll();
+    async handler(
```

Comment:
Add validation for query pagination values (we already have validation schema for it)